schema_salad/ref_resolver.py

```diff
@@ -936,6 +947,10 @@ def validate_links(self, document, base_url):
             iterator = enumerate(document)
         elif isinstance(document, dict):
             try:
+                for identifier in self.unique_ids:  # validates that all id's are unique
```

Comment:
Please use `self.identifiers` to enumerate the fields which represent identifiers.  The use of`self.unique_ids` is incorrect.